app/src/components/guild/index.tsx

```diff
@@ -0,0 +1,276 @@
+import { BigNumber } from 'ethers/utils'
+import React, { useEffect, useState } from 'react'
+import styled from 'styled-components'
+
+import { ConnectedWeb3Context } from '../../contexts'
+import { OmenGuildService } from '../../services/guild'
+import { getLogger } from '../../util/logger'
+import { RemoteData } from '../../util/remote_data'
+import { MarketFilters, MarketMakerDataItem, TransactionStep } from '../../util/types'
+import { Button } from '../button'
+import { ButtonType } from '../button/button_styling_types'
+import { IconBack } from '../common/icons'
+import { MarketCard } from '../market/market_card'
+import { ModalTransactionWrapper } from '../modal/modal_transaction'
+
+const logger = getLogger('Guild::Propose')
+
+interface Props {
+  count: number
+  context: ConnectedWeb3Context
+  currentFilter: any
+  isFiltering?: boolean
+  fetchMyMarkets: boolean
+  markets: RemoteData<MarketMakerDataItem[]>
+  moreMarkets: boolean
+  pageIndex: number
+  onFilterChange: (filter: MarketFilters) => void
+  onLoadNextPage: () => void
+  onLoadPrevPage: () => void
+}
+
+const GuildPageWrapper = styled.div`
+  width: 100%;
+  @media (max-width: ${props => props.theme.themeBreakPoints.xxl}) {
+    padding: 0 14px;
+  }
+`
+
+const ProposalHeadingWrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  align-items: center;
+  @media (max-width: ${props => props.theme.themeBreakPoints.sm}) {
+    flex-wrap: wrap;
+  }
+`
+
+const ProposalTitle = styled.div`
+  font-size: 22px;
+  font-weight: 500;
+  line-height: 26px;
+  color: ${props => props.theme.text3};
```

Comment:
Also for all the text files we should be using new TYPE.something from theme/index.js file It is now on master with recent pr from francesco that we pushed...

Reply:
done 🏄‍♂️ was cool putting `TYPE` into action, pretty powerful and easy to use